### ISSUE TITLE
documentation missing for multiple sub-domain updating

### ISSUE THREAD

**Issue by jahway603** [desec-io/desec-stack#587]
I am having trouble finding documentation to update multiple sub-domains on deSEC using the ddclient software, [as described here](https://desec.readthedocs.io/en/latest/dyndns/configure.html#option-2-use-ddclient)

I see a similar [Open issue here](https://github.com/desec-io/desec-stack/issues/187). I can't find documentation on how to do this anywhere. I've tried separating the sub-domains in the ddclient.conf as something like this, but can't get it working. It will update one host, but not all of them.

```
protocol=dyndns2
# "use=cmd" and the curl command is one way of doing this; other ways exist
use=cmd, cmd='curl https://checkipv4.dedyn.io/'
ssl=yes
server=update.dedyn.io

login=sub1.example.com
password='password'
sub1.example.com

login=sub2.example.com
password='password'
sub2.example.com
```

I've tried this with no go too

```
protocol=dyndns2
# "use=cmd" and the curl command is one way of doing this; other ways exist
use=cmd, cmd='curl https://checkipv4.dedyn.io/'
ssl=yes
server=update.dedyn.io

login=example.com
password='password'
sub1.example.com,sub2.example.com
```

**Comment by jahway603** [comment]
Update: my ddclient.conf contains each sub-domain out (as in the first code block) and it appears to have updated them all. Any idea if there is something that needs to be added to pause between them? Could I have been sending too much with the ddclient to the deSEC server?

**Comment by peterthomassen** [comment]
The [rate limit for dynamic IP updates is 1/min per domain](https://desec.readthedocs.io/en/latest/rate-limits.html). Independent domains can be updated at the same time.

As you already discovered, we currently don't support multi-domain update syntax (see #187).

However, I'm not fully clear yet whether you mean updating independent dynDNS domain (like a.dedyn.io and b.dedyn.io), or subdomains of your domain (e.g. sub1.c.dedyn.io and sub2.c.dedyn.io). Can you please clarify?

**Comment by jahway603** [comment]
Thank you for sharing the rate limit link. That links suggests creating CNAMES for each sub-domain, which I had not thought of,  so that might be the solution to my problem. I've been trying to update sub-domains of my domain (e.g. host1.mydomain.com, host2.mydomain.com) but there are a handful so I think I was hitting the rate limit. I'll try the CNAME redirects and report back.

**Comment by jahway603** [comment]
The CNAME redirects appear to be working, but I'm also not sure how to test if using CNAMEs instead of A records will mess up the SSL certs for each sub-domain.

**Comment by peterthomassen** [comment]
There's no reason why it should, TLS implementations don't care about how things are set up on the DNS level.

**Comment by jahway603** [comment]
Alright, thank you for clarifying this. I opened pull request #590 to help clarify this when someone goes looking for that info.

**Comment by jahway603** [comment]
This is resolved with PR #590. Thank you for incorporating it.
